pkg/local_workflows/depgraph_workflow.go

```diff
@@ -111,6 +111,10 @@ func depgraphWorkflowEntryPoint(invocation workflow.InvocationContext, input []w
 		debugLogger.Println("File:", file)
 	}
 
+	if debug := config.GetBool(configuration.DEBUG); debug {
```

Comment:
Why so complicated with assigning debug?

Reply:
made it shorter!